FSQCellManifest/FSQCellManifest.h

```diff
@@ -449,32 +450,32 @@ extern const NSInteger kRowIndexForFooterIndexPaths;
  
  @return A new table view cell manifest.
  */
-- (instancetype)initWithDelegate:(id)delegate 
-                         plugins:(NSArray *)plugins
+- (instancetype)initWithDelegate:(nullable id)delegate
```

Comment:
That won't necessarily be correct as this can also be UITableViewDelegate or UICollectionViewDelegate (it forwards along all the calls from UIKit if you implement them).